bigquery_etl/query_scheduling/templates/airflow_dag.j2

```diff
@@ -45,6 +45,10 @@ default_args = {{
 tags = {{ tags }}
 
 with DAG('{{ name }}', default_args=default_args{%+ if schedule_interval != None -%}, schedule_interval={{ schedule_interval | format_timedelta | format_schedule_interval }}{%+ endif -%}, doc_md = docs, tags = tags) as dag:
+{% for task_group in task_groups | sort %}
+    task_group_{{ task_group }} = TaskGroup('{{ task_group }}')
+{% endfor %}
+
```

Comment:
Any reason not to group tasks in each group under a context manager? Seems like that would help with readability

Reply:
I guess it was slightly easier with the generation logic